includes/degree-functions.php

```diff
@@ -333,13 +333,18 @@ function main_site_degree_search_program_types() {
 	ob_start();
 ?>
 	<div class="degree-search-types" ng-controller="ProgramController as programCtl" ng-init="programCtl.init()">
-		<h3 class="h5">Program Types</h3>
-		<div class="degree-search-type-container" ng-repeat="(key, type) in programCtl.programTypes">
-			<label class="form-check-label" ng-show="type.count > 0">
-				<input class="form-check-input" type="radio" name="program_type[]" value="{{ type.slug }}" ng-checked="mainCtl.selectedProgramType === type.slug" ng-click="programCtl.onSelected(type.slug)">
-				{{ type.name }} ({{ type.count }})
-			</label>
-		</div>
+		<h2 class="h4">Program Types</h2>
+		<a href="" ng-click="programCtl.onClear()">View All</a>
+		<ul class="degree-search-program-types list-unstyled">
+			<li class="degree-search-type" ng-repeat="(key, type) in programCtl.programTypes">
+				<a href="" ng-click="programCtl.onSelected(type.slug)">{{ type.name }}</a>
```

Comment:
You don't need the href="" here and on line 343.
https://code.angularjs.org/1.1.4/docs/api/ng.directive:ngHref